src/components/Main.js

```diff
@@ -1,13 +1,19 @@
 import React from "react";
-import Map from '../components/Map';
+import PropTypes from "prop-types";
+import Map from "../components/Map";
 
+const Main = (props) => {
+  const { data } = props;
 
-const Main = () =>{
-    return (
-        <section className="main">
-            <Map />
-        </section>
-    )
-}
+  return (
+    <section className="main">
+      <Map data={data} />
+    </section>
+  );
+};
 
-export default Main;
\ No newline at end of file
+Main.propTypes = {
+  data: PropTypes.object.isRequired,
```

Comment:
Nice type checking. Could you explain the advantage of using the prop-types package as opposed to just checking the type and throwing an error within the code? (We can talk about it on Sunday as well!)

Reply:
I'm not an expert on type checking, but I think the latter approach would be a little more cumbersome, as prop-types essentially does that for you and in a very concise, React friendly kind of way? Maybe Andrew can share a bit more insight / perspective on it.